### PR TITLE
Eclipse auto-generated code improvements

### PR DESCRIPTION
**Pull request by ckelleyRH**
Automagically done by Eclipse

### CODE REVIEW

File: src/main/java/com/netscape/management/client/util/Debug.java [hunk 1]
```diff
@@ -423,7 +423,8 @@ public static void println(String type, String s) {
                 System.err.println(s);
             }
             if (_fPrintCallStack) {
-                Thread.currentThread().dumpStack();
+                Thread.currentThread();
```
Comment (edewata) [inline src/main/java/com/netscape/management/client/util/Debug.java:426]: I suppose the `Thread.currentThread()` itself (and similar code below) is redundant.

**Review by edewata — APPROVED** [review]
LGTM, thanks! Just FYI we're planning to drop `pkiconsole` (including `idm-console-framework`) in PKI 11, so no need to spend too much time on this unless you're really bored :)

File: src/main/java/com/netscape/management/client/util/Debug.java (hunk 1 above)
Comment (ckelleyRH) [reply src/main/java/com/netscape/management/client/util/Debug.java:426]: Indeed, I imagine there is plenty to scrape out if one were to look deep into the abyss!

**Comment by ckelleyRH** [comment]
OK, if this is marked for exclusion then I'll focus any future janitorial efforts elsewhere! I was mainly doing it to pull all the dependent repos and see what was in there/how everything fits together.

**Pull request merged**